client/components/gravatar/test/index.jsx

```diff
@@ -1,96 +1,103 @@
 /**
  * External dependencies
  */
-import assert from 'assert';
-import ReactDomServer from 'react-dom/server';
+import { expect } from 'chai';
 import React from 'react';
+import { shallow } from 'enzyme';
 
 /**
  * Internal dependencies
  */
 import { Gravatar } from '../';
 
-/**
- * Pass in a react-generated html string to remove react-specific attributes
- * to make it easier to compare to expected html structure
- * @param {string} string react-generated html
- * @return {string} HTML without react attributes
- */
-function stripReactAttributes( string ) {
-	return string.replace( /\sdata\-(reactid|react\-checksum)\=\"[^\"]+\"/g, '' );
-}
-
-describe( 'Gravatar', function() {
-	const bobTester = {
-		avatar_URL: 'https://0.gravatar.com/avatar/cf55adb1a5146c0a11a808bce7842f7b?s=96&d=identicon',
+describe( 'Gravatar', () => {
+	const genericUser = {
+		avatar_URL: 'https://0.gravatar.com/' +
+			'avatar/cf55adb1a5146c0a11a808bce7842f7b?s=96&d=mm',
 		display_name: 'Bob The Tester'
 	};
```

Comment:
one thing I strongly encourage (and maybe it's too much of a change with existing use of Gravatar or at minimum would be a PR in and of itself) is to try and not pass down objects as props to React components. if it's a number, boolean, string, or list, pass them in, but if an object, pass a simple reference and get the object from the state tree.

this could have a positive effect here.

anyways, feel free to move along with this PR